examples/twittermap/web/public/javascripts/lib/hashmap.js

```diff
@@ -0,0 +1,210 @@
+/**

```

Comment:
we should remove this file.

Reply:
Got it. How can I make this happen? Check out her branch and remove this file and then push the change to her branch? Is this possible?

Reply:
I will request a new PR.